parts/todo/regen.out

```diff
@@ -0,0 +1 @@
+Can't open perl script "devel/regenerate": No such file or directory
```

Comment:
Do you really want to merge this file?

Reply:
As I said, it is not ready for merge.  I need to reorder some commits, and go through them to find flaws like this

Reply:
great news, thanks @khwilliamson for the update